python/lsst/faro/measurement/DetectorTableMeasurement.py

```diff
@@ -61,10 +61,10 @@ class DetectorTableMeasurementConfig(
 
     def validate(self):
         super().validate()
-        if "detector" not in self.columns:
+        if "detector" not in self.measure.columns:
```

Comment:
I'm having a hard time figuring out how this is related.  Should it really be a separate bugfix commit?

Reply:
Thanks. I'll make that a separate commit.